pkg/controller/pod.go

```diff
@@ -243,6 +243,14 @@ func (pnc *PodNetworksController) handlePodUpdate(oldObj interface{}, newObj int
 	oldPod := oldObj.(*corev1.Pod)
 	newPod := newObj.(*corev1.Pod)
 
+	if newPod.Spec.HostNetwork {
+		klog.Warningf(
+			"rejecting to add interfaces for host networked pod: %s",
+			annotations.NamespacedName(newPod.GetNamespace(), newPod.GetName()),
+		)
+		pnc.Eventf(newPod, corev1.EventTypeWarning, "InterfaceAddRejected", rejectInterfaceAddEventFormat(newPod))
```

Comment:
I'm assuming we want to both throw an event an log ... Maybe that's too much.

Any thoughts @kmabda ?

Reply:
I think it's a good idea to throw an event here to give a feedback to the user

Reply:
One thing, since in https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller/pull/90/files#diff-449e8d1bd46b9c978e208dde912a1d7a76d17ce553067f8d0a67a27f17338d26R152 we need to filter out hostnetwork pods as well but we can't tel before hand if an interface got added by the user or not until we process the request we could move this logic from `handlePodUpdate` to `processNextWorkItem` function.

**Pro** same treatment logic would be in one place i.e. we have a consistency w.r.t. filtering out unwanted pods and how we log/throw out an event to the user in case an interface got added for these pods in one place
**Con** a little more overhead by adding an item to the workqueue than removing it for processing

If you go ahead with these changes I can rebase my pull request once this one get merged.